tests/data/pytorch_eia/mnist.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
would it make more sense to name this file something like "empty_inference_script"? imo "mnist" implies a script that trains a model on mnist because that's how we use it everywhere else :joy:

Reply:
updated script name.